app/views/patients/lookup.html.erb

```diff
@@ -0,0 +1,68 @@
+<h1>Lookup Patients</h1>
+
+<p class="info">
+  <i class="fas fa-question-circle"></i>
+  This is a hidden page and is only visible to you because you have the
+  <em><%= Permissions::ALL_PERMISSIONS.dig(:view_overdue_list, :slug) %></em>
+  permission.
```

Comment:
Who is this messaging for? If it's for our alpha users (a few CVHOs), should it instead describe something like "Alpha", " You can search for your patients by village and street here", "Work in Progress", etc? Let's ask someone about the right message for this bit.

Reply:
Consulted with Vineet and modified it a bit.